lib/Driver/ToolChains.cpp

```diff
@@ -1104,6 +1113,16 @@ getSanitizerRuntimeLibNameForDarwin(StringRef Sanitizer,
       + (shared ? "_dynamic.dylib" : ".a")).str();
 }
 
+static std::string
+getSanitizerRuntimeLibNameForWindows(StringRef Sanitizer,
+                                    const llvm::Triple &Triple,
+                                    bool shared = true) {
+  return (Twine("clang_rt.")
+      + Sanitizer + "-"
+      + Triple.getArchName()
+      + (shared ? ".dll" : ".lib")).str();
```

Comment:
I'm not sure that this is right.  You never link against the `.dll` but rather the `.lib`.